src/utils/body.ts

```diff
@@ -92,7 +92,10 @@ export function readRawBody<E extends Encoding = "utf8">(
       }
       // TODO: Handle other BodyInit types
       // https://developer.mozilla.org/en-US/docs/Web/API/Response/Response#body
-      if (_resolved instanceof URLSearchParams) {
+      if (
+        _resolved instanceof URLSearchParams ||
+        _resolved instanceof FormData
+      ) {
         return Buffer.from(_resolved.toString());
```

Comment:
`FormData.toString` returns `[object FormData]`. 

`await new Response(data).bytes()` can be used to normalize

Reply:
Oh, sorry, I missed that. Corrected 👍

Reply:
seems ci is failing can you check

Reply:
Now it works 😃